src/main/java/org/folio/rest/impl/PatronBlockConditionsAPI.java

```diff
@@ -0,0 +1,68 @@
+package org.folio.rest.impl;
+
+import java.util.Map;
+
+import javax.ws.rs.core.Response;
+
+import org.folio.rest.jaxrs.model.PatronBlockCondition;
+import org.folio.rest.jaxrs.resource.PatronBlockConditions;
+import org.folio.rest.persist.PgUtil;
+
+import io.vertx.core.AsyncResult;
+import io.vertx.core.Context;
+import io.vertx.core.Handler;
+
+public class PatronBlockConditionsAPI implements PatronBlockConditions {
+
+  private static final String PATRON_BLOCK_CONDITIONS = "patron_block_conditions";
+
+  @Override
+  public void getPatronBlockConditions(int offset, int limit, String query,
```

Comment:
Can you add the `@Validate` annotation to all methods so that RMB validates the input data before calling the method?

Reply:
added